ironfish/src/metrics/metricsMonitor.ts

```diff
@@ -78,4 +90,22 @@ export class MetricsMonitor {
     this.heapUsed.value = memoryUsage.heapUsed
     this.rss.value = memoryUsage.rss
   }
+
+  private submitMemoryTelemetry(): void {
+    submitMetric({
+      name: 'memory',
+      fields: [
+        {
+          name: 'heap_used',
+          type: 'integer',
+          value: this.heapUsed.value,
+        },
+        {
+          name: 'heap_total',
+          type: 'integer',
+          value: this.heapTotal.value,
+        },
+      ],
+    })
+  }
```

Comment:
If I was trying to find the place where telemetry is sent to the web, I would look in the telemetry system. How do you feel about moving this after your refactor of the telemetry system?

Reply:
Yeah, that's the plan. This was mostly to keep the diff focused on uploading heap stats.